logging/google/cloud/logging/handlers/app_engine.py

```diff
@@ -67,3 +69,17 @@ def get_gae_resource(self):
             },
         )
         return gae_resource
+
+    def get_gae_labels(self):
+        """Return the labels for GAE app which includes trace_id.
+
+        :rtype: dict
+        :returns: Labels for GAE app.
+        """
+        trace_id = get_trace_id_from_request_header()
+        if trace_id is None:
+            trace_id = 'unknown'
```

Comment:
The log aggregation is correlating the app logs to request log, and the trace_id for request log is always valid (or cannot be 'unknown'?). So I think it will not pool a bunch of stuff together. I have tried using None as the value, but the field does not allow None type. Or maybe to avoid this, just set the labels filed to None if trace_id is None?